versions/3.0.4.md

```diff
@@ -3034,11 +3036,16 @@ Field Name | Type | Description
 
 ##### Conditions for Using the Discriminator Object
 The Discriminator Object is legal only when using one of the composite keywords `oneOf`, `anyOf`, `allOf`.
```

Comment:
Does this mean that the `Pet` example right above is illegal because it uses `discriminator` and lacks

```yaml
      anyOf:
      - $ref: '#/components/schemas/Cat'
      - $ref: '#/components/schemas/Dog'
```

???

Reply:
No, that's explained in the next few paragraphs.  Also, this is not part of the change in this PR.

Reply:
Found it, now that I know it is there 😄